cmd/kops/create_cluster.go

```diff
@@ -86,6 +87,8 @@ func init() {
 	cmd.Flags().StringVar(&createCluster.AdminAccess, "admin-access", "", "Restrict access to admin endpoints (SSH, HTTPS) to this CIDR.  If not set, access will not be restricted by IP.")
 
 	cmd.Flags().BoolVar(&createCluster.AssociatePublicIP, "associate-public-ip", true, "Specify --associate-public-ip=[true|false] to enable/disable association of public IP for master ASG and nodes. Default is 'true'.")
+
+	cmd.Flags().StringVarP(&createCluster.Manifest, "manifest", "f", "", "Optional .yml cluster manifest")
```

Comment:
I would steal the verbiage from `kubectl`: -f, 

We are not going to do a directory at this point. 
```
--filename=[]: Filename, directory, or URL to a file identifying the resource to get from a server.
```